_assets/stylesheets/pages/_onsite-group.scss

```diff
@@ -0,0 +1,19 @@
+.card-title--overlap-remove-divider {
+  background-color: $cr-white;
+  display: inline-block;
+  font-family: $condensed-extra-font-face;
+  font-size: 2.125rem;
+  margin: -2rem -0.5rem 0;
+  padding: 0.25rem 0.5rem;
+  position: relative;
+}
+
+.card-title--overlap-remove-divider:after {
+  color: transparent;
+  content: '\2014';
+  display: block;
+  font-family: $base-font-face;
+  font-size: 1.25rem;
+  font-weight: 100;
+  line-height: 1.5;
+}
```

Comment:
🚫 on GitHub means the file is missing a final new line.

If you don't want to manually add a new line to every file you edit, let VS Code do for you. 😄 

![image](https://user-images.githubusercontent.com/1730906/63292465-b12ce000-c293-11e9-8961-7b09cd3c2747.png)